core/src/main/java/org/lflang/generator/c/CWatchdogGenerator.java

```diff
@@ -9,16 +9,12 @@
 package org.lflang.generator.c;
 
 import java.util.List;
+
 import org.lflang.MessageReporter;
 import org.lflang.ast.ASTUtils;
 import org.lflang.generator.CodeBuilder;
 import org.lflang.generator.ReactorInstance;
-import org.lflang.lf.Mode;
-import org.lflang.lf.ModeTransition;
-import org.lflang.lf.Reactor;
-import org.lflang.lf.VarRef;
-import org.lflang.lf.Variable;
-import org.lflang.lf.Watchdog;
+import org.lflang.lf.*;
```

Comment:
The style guide does not allow wildcard imports.

Reply:
Intellij does this automatically.  Here are the places where this appears, most of which have nothing to do with this PR:

<img width="1108" alt="image" src="https://github.com/user-attachments/assets/d8e718ed-0d60-4c13-842f-9129045e75f7">

Reply:
It must have slipped past reviews before, but that doesn't change the fact that the style guide prohibits it... Doesn't IntelliJ ask for confirmation before it carries out such automated refactoring?